src/offspot_demo/constants.py

```diff
@@ -1,10 +1,29 @@
+import os
 from pathlib import Path
 
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
 OFFSPOT_IMAGE_ID = "offspot-demo"
-OFFSPOT_IMAGE_URL = "https://api.imager.kiwix.org/auto-images/{OFFSPOT_IMAGE_ID}/json"
+OFFSPOT_IMAGE_URL = f"https://api.imager.kiwix.org/auto-images/{OFFSPOT_IMAGE_ID}/json"
 
-TARGET_DIR = Path("/data")
-IMAGE_PATH = Path("/demo/image.img")
+TARGET_DIR = Path("/host/data")
+IMAGE_PATH = Path("/host/demo/image.img")
 PREPARED_FLAG_PATH = TARGET_DIR / "prepared.ok"
 
-FQDN = "demo.hostpot.kiwix.org"
+FQDN = os.getenv("OFFSPOT_DEMO_FQDN", "demo.hostpot.kiwix.org")
+
+DOCKER_COMPOSE_PROD_PATH = TARGET_DIR / "compose.yaml"
```

Comment:
Can we use the `maint` and `image` names? `prod` is unclear as maintenance is still in prod 😉

Reply:
If find `image` just as confusing in my mind, but I agree `prod` is not better, so I don't mind to change to this if it is easier at least for you 😄